src/app/activityintro/activityintro.component.scss

```diff
@@ -0,0 +1,11 @@
+.introbody{
+	text-align: center;
```

Comment:
spacing.

also, i'm seeing a lots of classes with `text-align: center`, maybe it's time to consider creating a class for it? or even considering existing [bootstrap class](http://getbootstrap.com/css/#helper-classes-center)